log_bucket/README.md

```diff
@@ -0,0 +1,42 @@
+# log_bucket
+
+This module was inspired by https://github.com/jetbrains-infra/terraform-aws-s3-bucket-for-logs
+
+Note: be sure to use the desired branch/tagged release.
+
+Usage:
+
+```terraform
```

Comment:
Are you going to include in this PR updates to the different buckets, CloudFronts, and LBs to point to these?